### DISCUSSION TITLE
Skippable Pre-missions

### DISCUSSION THREAD

**Discussion by grztx** [YimMenu/YimMenu#1044] [Ideas]
It would be cool if the menu would have functions to skip for example the Casino Heist Prebs and Cayo Perico Prebs

**Comment by Nokkel4** [comment]
What happened prometheus discord server? 
Is there new discord server already? 
If yea send me invite 
My discord: Nokkela Vihtori#2573

**Reply by Yimura** [reply]
Does this look like Prometheus support?

**Reply by LiamD-Flop** [reply]
Does this look like Prometheus support?

**Comment by Nokkel4** [comment]
Well idk where to get some answers prometheus used this base

**Reply by Yimura** [reply]
That does not mean we are affiliated with Prometheus please go look somewhere else.
